modules/salt/nav-salt-guide.adoc

```diff
@@ -17,6 +17,9 @@ endif::[]
 ** xref:salt-file-locations.adoc[File Locations]
 ** xref:salt-gitfs.adoc[GitFS Fileserver Backend]
 ** xref:yomi.adoc[Install Using Yomi]
+ifeval::[{uyuni-content} == true]
```

Comment:
This is going to go into 4.1.2 as technical preview, so this content is not Uyuni specific.

Reply:
OK, let's remove the ifeval, then. I was confused.